package.json

```diff
@@ -645,6 +695,47 @@
                 "title": "%AWS.command.hideRegion%",
                 "category": "AWS"
             },
+            {
+                "command": "aws.s3.copyPath",
+                "title": "%AWS.command.s3.copyPath%",
+                "category": "AWS"
+            },
+            {
+                "command": "aws.s3.downloadFileAs",
+                "title": "%AWS.command.s3.downloadFileAs%",
+                "category": "AWS",
+                "icon": {
+                    "light": "third-party/resources/from-vscode-icons/light/cloud-download.svg",
+                    "dark": "third-party/resources/from-vscode-icons/dark/cloud-download.svg"
+                }
+            },
+            {
+                "command": "aws.s3.uploadFile",
+                "title": "%AWS.command.s3.uploadFile%",
+                "category": "AWS",
+                "icon": {
+                    "light": "third-party/resources/from-vscode-icons/light/cloud-upload.svg",
+                    "dark": "third-party/resources/from-vscode-icons/dark/cloud-upload.svg"
+                }
+            },
+            {
+                "command": "aws.s3.createFolder",
+                "title": "%AWS.command.s3.createFolder%",
+                "category": "AWS",
+                "icon": {
+                    "light": "third-party/resources/from-vscode-icons/light/new-folder.svg",
+                    "dark": "third-party/resources/from-vscode-icons/dark/new-folder.svg"
+                }
+            },
+            {
+                "command": "aws.s3.createBucket",
+                "title": "%AWS.command.s3.createBucket%",
+                "category": "AWS",
+                "icon": {
```

Comment:
Could there be a "bucket with a plus symbol" to better represent "Create bucket" ?

Reply:
Yes, that's one of the things I wanted to get a new icon for. What do you think about the download and upload icons (cloud with up or down arrow)? I kind of think that should be the doc with an up or down arrow. 

I'm noticing now that GitHub actually uses a similar cloud with an up arrow for uploading files to this comment box haha.

Reply:
Added the create bucket icon and kept the cloud icons for now